src/main/java/com/github/lunatrius/schematica/proxy/ClientProxy.java

```diff
@@ -247,7 +249,7 @@ public static boolean addCoordinates(String worldServerName, String schematicNam
      * @return {@link ImmutablePair} with bool (true if coordinates found, false if not) and {@link ImmutableTriple}
```

Comment:
This is now outdated as you replaced it with a Triple

Reply:
I'm a dummy at docs :) 
I have pushed a change, feel free to let me know if it needs further changes.